src/components/pages/Home.tsx

```diff
@@ -117,6 +129,16 @@ export default function HomePage(): ReactElement {
           </Button>
         }
       />
+
+      <SectionQueryResult
+        title="New Algorithms"
```

Comment:
we should get in latest changes from `feature/compute` which itself includes latest changes from `main` where we have some changed labeling on front-page so however we name it here needs to be considered in that context.

Then, if we have a section with new algorithms only, it implies that the section above only shows latest data sets, but no algorithms.

Not sure about the Highest Liquidity section yet if we want to mix them there, or rather separate. Likewise, I'm thinking about some visual separation cause right now we visually separate between download/compute datasets but not between datasets/algorithms. On top of that for algorithms, pretty sure we do not want to have a `Compute` label show up on their `AssetTeaser` despite them having a compute service

Reply:
maybe we can keep only the sections "Highest Liquidity" and "Recently Published" which contains both algorithms and data sets and after sorting by service type is implemented on the search page we can change the links from sections by replacing `DATA SETS` with `ASSETS`. Then the user can do the sorting o search page

Reply:
I just have merged the changes from `feature/compute` in this branch so now it's up to date .

On the datasets section I  added a filter, so now we can have a clear separation between datasets and algorithms sections.

Maybe separating Highest Liquidity Datasets / Algorithms in two sections would result in way too many sections on market page. A visual separation between algorithms and datasets would make things more clear for user, but I'm not sure how we could achieve that visually, without crowding the design.

Reply:
yup, keeps front-page more simple. But requires some visual distinction, currently trying to come up with some quick visual system accounting for download/compute/dataset/algorithm. If we have that we can show them all mixed everywhere by default

Reply:
Aaaaand we arrived back at this after some global color tinkering in 2427c8e. We can now: 

- mix both types in all sections on front page
- remove this _New Algorithms_ section. But might be handy to leave it in for now, but remove it before final release of everything